lib/node_modules/@stdlib/stats/base/dvarmpn/benchmark/c/benchmark.length.c

```diff
@@ -120,6 +121,40 @@ static double benchmark( int iterations, int len ) {
 	return elapsed;
 }
 
+/**
+* Runs a benchmark.
+*
+* @param iterations   number of iterations
+* @param len          array length
+* @return             elapsed time in seconds
+*/
+static double benchmark2( int iterations, int len ) {
+	double elapsed;
+	double x[ len ];
+	float v;
+	double t;
+	int i;
+
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = ( rand_double() * 20000.0 ) - 10000.0;
+	}
+	v = 0.0f;
+	t = tic();
+	for ( i = 0; i < iterations; i++ ) {
+		// cppcheck-suppress uninitvar
+		v = stdlib_strided_dvarmpn_ndarray( len, 0.0, 1, x, 1, 1 );
```

Comment:
```suggestion
		v = stdlib_strided_dvarmpn_ndarray( len, 0.0, 1, x, 1, 0 );
```
just following the convention here